pom.xml

```
@@ -30,12 +30,19 @@
         </sonar.coverage.jacoco.xmlReportPaths>
         <sonar.exclusions>
             **/uz/enterprise/mytex/MytexApplication.java,
-            **/uz/enterprise/mytex/config/**/*,
+            **/uz/enterprise/mytex/configs/**/*,
```

```suggestion
            **/uz/enterprise/mytex/config/**/*,
```
use singular nouns for packages